mender-convert

```diff
@@ -91,15 +108,23 @@ if [ -z "${MENDER_ARTIFACT_NAME}" ]; then
   echo -e "\tMENDER_ARTIFACT_NAME=\"release-1\" ./mender-convert"
   exit 1
 fi
-
-source modules/bootstrap.sh
-
 mkdir -p work
 touch work/convert.log
 
-./mender-convert-extract "$@"
-./mender-convert-modify "$@"
-./mender-convert-package "$@"
+parse_cli_options "$@"
+
+uncompressed_disk_image="${disk_image}"
+compression_type=$(compression_type "${disk_image}")
+if [[ ${compression_type} != "none" ]]; then
+  uncompressed_disk_image=$(decompress_image "${disk_image}")
+  tfile=$(mktemp "mender-convert-compress-configuration.XXXXX") || log_fatal "Tempfile creation failed"
```

Comment:
Oh here I see it is stored in `/tmp`, I guess it makes sense.

Why the `XXXXX` ?

I do see other usage areas for this `tmp` file so could keep the name of the file generic. But this is just a nitpick,

Reply:
The `XXXX` is just where to put the random numbers.

But I actually liked ur idea of simply putting it in work better actually, so I think I'll just drop the whole tmpfile thing, and store it in work, unless you object.

Reply:
No objections